### ISSUE TITLE
Search By Name> Group Radio Button

### ISSUE THREAD

**Issue by KendraStraub** [nationalparkservice/npmap-species#55]
Search area allows users to use radio button to select either species or group-- but when group is selected, the same functionality exists in the search bar-- species names appear. The help text ("Type a species name") should also change when group is selected. 

I think we discussed a while back that the "group" search would be a drop down, as the term "group" is something we chose arbitrarily and users will have no idea what we mean by it and therefore won't know what to type in that bar. 

![3](https://cloud.githubusercontent.com/assets/8036575/8546430/b2ad618a-2485-11e5-9dc5-72971a8f6d08.PNG)

**Comment by jduggan1** [comment]
The way it currently works is for group search to mean that you search for a species by name, and then the result is the group that it belongs to.  We have about 100 groups now, so a dropdown isn't a good fit.

This might fit better under the "need to add better explanations" issue which you raised a few weeks ago. I'll add Sarah to this and see what she thinks.

**Comment by KendraStraub** [comment]
Thanks John-- We have narrowed the taxa group list to 24-- is that still too long for a drop down?

**Comment by jduggan1** [comment]
I'm not using the taxa groups.  We had a problem with the taxa groups containing too many species to use for this, so instead I created the groups myself and limited the number of species per group (I want to say I limited it to 15 species per group).  However, in doing that, the number of groups blows up necessarily.

This is probably one that Sarah will need to examine, not sure what she would recommend here.
